.github/invite-contributors.yml

```diff
@@ -1,4 +0,0 @@
-# invite-contributors config
```

Comment:
This needs fixing rather than removing. It should automatically add those who contribute regularly to the contributor team, but would need @Robbt or @hairmare to fix

Reply:
I don't really see why we should automatically add contributors as members.

It should be done by hand, and you should have permissions to add new members.

Reply:
I'll open a ticket and remove the commit for the PR, so we can discuss this properly.